dragon.c

```diff
@@ -508,6 +522,18 @@ int main (int argc, char **argv) {
         } else if (strcmp(argv[i], "-I") == 0
                 || strcmp(argv[i], "--stdin") == 0) {
             from_stdin = true;
+        } else if (strcmp(argv[i], "--targets") == 0) {
+            mode = MODE_TARGET;
+            targets = &argv[i+1];
+            while (argv[i+1] != NULL && strncmp(argv[i+1], "-", 1) != 0) {
```

Comment:
I don't use this feature so I can't judge how useful it is, but here's a small nit: using `strncmp` to just test a single char is kinda silly. Just check it with `!=`.

```suggestion
            while (argv[i+1] != NULL && argv[i+1][0] != '-') {
```

Reply:
Yes of course, I don't know what I did there. Thank you.